roles/davinci_resolve/tasks/main.yml

```diff
@@ -129,6 +137,7 @@
   ansible.builtin.file:
     path: "/tmp/{{ product | replace(' ', '_') }}_{{ pkgver }}.zip"
     state: absent
+  when: davinci_resolve_zip_file is undefined
```

Comment:
If you are making a copy of the file, why is the copy not getting removed once is not needed anymore?